src/features/navigation/RootNavigator/useInitialScreenConfig.tsx

```diff
@@ -12,46 +12,33 @@ import { homeNavConfig } from '../TabBar/helpers'
 import { RootScreenNames } from './types'
 
 export function useInitialScreen(): RootScreenNames | undefined {
-  const { isLoggedIn } = useAuthContext()
-
+  const { data: user } = useUserProfileInfo()
   const [initialScreen, setInitialScreen] = useSafeState<RootScreenNames | undefined>(undefined)
 
   useEffect(() => {
-    getInitialScreen({ isLoggedIn }).then((screen) => {
-      setInitialScreen(screen)
-      triggerInitialScreenNameAnalytics(screen)
-    })
-  }, [isLoggedIn])
+    getInitialScreen(user)
+      .then((screen) => {
+        setInitialScreen(screen)
+        triggerInitialScreenNameAnalytics(screen)
+      })
+      .catch(() => {
+        // If we cannot read into the storage, we just go to the homepage
```

Comment:
Vu que tu as déplacé le catch le commentaire précédent n'est plus assez général : 
```suggestion
        // If we cannot resolve an initial screen, we just go to the homepage
```

Reply:
J'avais mis cette logique initialement car react-navigation ne peut accepté que un seul initial screen (pas de rerender possible), donc fallait être certain qu'on avait la bonne info dés le début, sans se baser sur les mises à jours de state ou autre